app/views/layouts/_head.html.erb

```diff
@@ -78,6 +78,13 @@
     document.addEventListener('click', _earlyClick)
   </script>
   <%= javascript_include_tag(active_brand_config_js_url, defer: true) %>
+
+  <!-- TODO: StrongMind - Remove me? -->
+  <script
```

Comment:
We probably don't want to load JQuery on all pages.  Can we move it to the page in a content for?